content/blog/most-used-keyboard-shortcuts.md

```diff
@@ -0,0 +1,90 @@
+---
+author: "Alex Hill"
+date: 2020-01-02
+title: My most used keyboard shortcuts
+---
+
+Anyone who uses a computer for a significant amount of time each day should 
+spend some time learning keyboard shortcuts - they can save you so much time! 
+
+When I started working as a developer, I learnt keyboard shortcuts through pair programming, 
+asking more experienced programmers what their favourite shortcuts were, and just reading
+documentation for the IDE (Intelligent Development Editor) I was using 
+([VisualStudio](https://docs.microsoft.com/en-us/visualstudio/ide/default-keyboard-shortcuts-in-visual-studio?view=vs-2019) 
+with [Resharper](https://www.jetbrains.com/help/resharper/Reference__Keyboard_Shortcuts.html) at the time,
+ now [IntelliJ](https://www.jetbrains.com/help/idea/mastering-keyboard-shortcuts.html).) 
+
+Here are my most used keyboard shortcuts, starting with the basics:
+
+### Global
+These shortcuts are absolute essentials and work in most apps (text editors, browsers, email clients, IDEs, etc.)
+ If you don't already use them, spend some time memorising and practising them. It'll save you loads of time in the long run.
+
+* `ctrl-v` - copy
```

Comment:
throughout, if you use `<kbd>ctrl</kbd>` you'll get a little keyboardy markup like <kbd>ctrl</kbd>

Reply:
Yesss thank you! I was wondering how to make them look like that!